FEATURES.md

```diff
@@ -6,11 +6,10 @@ You can read the full reference for both of them [here](docs/syntax.md), but the
 To see some more complex examples programs, check out the [examples](examples/) folder.
 
 ### Basic features
-
-We can start with a basic program that adds the numbers 3 and 2.
+Types in Bend are completely optional - you can write programs without any type annotations, but we'll be typing every function for clarity. We can start with a basic program that adds the numbers 3 and 2. Though 
```

Comment:
This is too complicated for first example. Let's try breaking into 2 pieces, one just like the old one and then another showing optional typing.